frontend/lib/ui/artefact_page/test_execution_review.dart

```diff
@@ -49,7 +49,7 @@ class TestExecutionReviewButton extends StatelessWidget {
           ),
           direction: PopoverDirection.bottom,
           width: 500,
-          height: 500,
+          height: 400,
```

Comment:
The PopOver library doesn't support automatic sizing, so I have adapted the size to fit with less choices.

Reply:
What happens if there were deprecated options selected? would the user have to scroll in the popover? can't say I like that

Reply:
Yeah, the user would have to scroll a bit. I couldn't find a way to add some dynamic sizing to this widget. Perhaps the other alternative would be to count those and add some sizing based on how many we should show, but not sure how nice that would look in code.

Reply:
Yeah the alternative is not great either. This package isn't very flexible. I guess we can keep the scroll